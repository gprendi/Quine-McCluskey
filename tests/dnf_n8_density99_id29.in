d 8
00100111
11100001
01110100
00010101
10110110
00110000
00000110
10010001
10000110
10101010
11101011
10010100
00101100
11001000
01001011
01000000
01110010
00010000
01000101
11011101
01110111
10011110
00111000
00111110
01000100
00010111
00111001
01001001
10110010
00101001
11000110
00100110
01000001
00101010
00011011
11101001
11011010
01100100
10111111
01110001
01000010
11110011
01111100
01101111
00101011
01110101
10010010
00011010
11100101
01100101
00100000
10100111
01010101
11110010
11100100
00001100
01010110
00111100
01100000
11001111
10001011
11100010
10100001
11110100
01101110
01011001
10000111
00011110
11100000
01001110
01110000
00001010
10101001
00100010
01101010
00111111
00001001
10001100
10111110
01111101
01011101
10000100
10101011
11110110
10110001
00110010
00100101
10111011
11010000
01001100
11000100
11101010
00011111
11111111
11001011
00111011
11111100
10110000
01001000
01011011
01111110
11010011
11111101
11010101
11110111
01101011
11001010
11100110
11100011
10101111
01110011
10000101
00001111
10011011
10111100
10000010
00110011
01100001
00111010
10000000
01010001
00110101
10001010
01111111
01011111
00010001
10001101
10101110
10010000
11101101
11011011
11011111
01101100
11010111
01111001
01010011
11110000
10100100
10001000
01111000
00000100
11100111
10011000
01011110
11000001
10110011
11001001
01001010
00011001
10110100
10111000
11000010
01010010
11010010
01101001
00101110
11001101
11110001
00011000
01101000
11010100
10010101
00001011
01010111
11011000
10011100
01001111
11101100
11111011
11001100
10011101
00000011
11111110
00101111
01100010
10101100
01011100
00110111
11101000
00110110
10100110
11101111
01101101
00000001
10010110
10111010
10010111
01010100
01110110
00000000
00011101
01001101
01100011
10011001
10011111
10100011
01100110
11111001
01010000
00100001
10100000
11111000
11000000
00100011
10001001
11010110
10110111
00101101
00101000
11011001
11101110
01011010
10110101
01111010
00001110
01011000
11010001
00001101
11011110
00001000
11000011
10001111
00000101
11000111
01111011
10001110
00000010
11011100
10101101
00010011
01000110
11111010
00111101
00010100
00110100
10100010
00011100
10111001
00110001
00010110
00100100
00000111
10010011
10100101
10101000
10111101
00010010
10011010
11110101
01100111
01000111
01000011
10000011
