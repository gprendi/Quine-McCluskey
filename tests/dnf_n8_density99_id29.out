p dnf 8 0
-1 0
-2 5 0
-2 6 0
-2 7 0
-2 -8 0
2 -6 0
3 0
4 0
-5 7 0
-5 -8 0
5 -6 0
5 -7 0
5 8 0
-6 7 0
-6 -8 0
-7 -8 0
7 8 0
